src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -757,6 +757,65 @@ bool LibraryCallKit::inline_vector_shuffle_to_vector() {
   return true;
 }
 
+// public static
+// <E,
+//  SH extends VectorShuffle<E>>
+// SH wrapShuffleIndexes(Class<E> eClass, Class<? extends SH> shClass, SH sh, int length,
+//                       ShuffleWrapIndexesOperation<SH> defaultImpl)
+bool LibraryCallKit::inline_vector_wrap_shuffle_indexes() {
+  const TypeInstPtr* elem_klass    = gvn().type(argument(0))->isa_instptr();
+  const TypeInstPtr* shuffle_klass = gvn().type(argument(1))->isa_instptr();
+  Node*              shuffle       = argument(2);
+  const TypeInt*     vlen          = gvn().type(argument(3))->isa_int();
+
+  if (elem_klass == nullptr || shuffle_klass == nullptr || shuffle->is_top() || vlen == nullptr) {
+    return false; // dead code
```

Comment:
Why dead code  in comment ? this is a failed intrinsification condition.

Reply:
Modified comment.